src/Shopsys/ShopBundle/Resources/config/services.yml

```diff
@@ -182,4 +186,3 @@ services:
     Shopsys\ShopBundle\Model\AdvancedSearch\ProductAdvancedSearchConfig: ~
 
     Shopsys\FrameworkBundle\Model\AdvancedSearch\ProductAdvancedSearchConfig: '@Shopsys\ShopBundle\Model\AdvancedSearch\ProductAdvancedSearchConfig'
-
```

Comment:
```suggestion

```

Reply:
There were two newlines at the end. It's hard to see on Github web UI